### DISCUSSION TITLE
Add weeknumber on the calendar view

### DISCUSSION THREAD

**Discussion by tuytra** [totaldebug/atomic-calendar-revive#1453] [Feature Requests]
Could you make weeknumber on the calendar view at beginning of the week. Like the example under ?

<img width="571" alt="Skjermbilde 2024-03-18 kl  22 04 40" src="https://github.com/totaldebug/atomic-calendar-revive/assets/1047386/f59de2cf-2ec5-4907-a1b3-50ffabefe532">

**Comment by marksie1988** [comment]
This has now been added to the latest beta, will be released shortly
